api/services/search.service.js

```diff
@@ -0,0 +1,16 @@
+import { InvalidData } from "../errors/InvalidData.js";
+import * as searchRepository from "../repositories/search.repository.js";
+export const getProductsAndStores = async (req, res, next) => {
+    var { query } = req.body;
+    if(query.length < 3){
```

Comment:
cant u do this using validation rules? u might need to create a custom 1.

Reply:
Ok